etc/kayobe/environments/ci-multinode/kolla.yml

```diff
@@ -5,7 +5,15 @@ kolla_enable_neutron_provider_networks: true
 kolla_enable_ovn: true
 kolla_enable_octavia: true
 kolla_enable_magnum: true
+kolla_enable_designate: true
+kolla_enable_redis: true
 
 # The multinode environment supports Manila but it is not enabled by default.
 # kolla_enable_manila: true
 # kolla_enable_manila_backend_cephfs_native: true
+
+kolla_internal_vip_address: "{{ internal_vip_address }}"
+kolla_internal_fqdn: "{{ internal_fqdn }}"
+
+kolla_external_vip_address: "{{ public_vip_address }}"
+kolla_external_fqdn: "{{ public_fqdn }}"
```

Comment:
These variables are the newer alternatives to the <net>_vip_address and <net>_fqdn variables. If you want to use these it would be better to remove the others from networks.yml